packages/@react-spectrum/overlays/src/Overlay.tsx

```diff
@@ -42,8 +42,11 @@ function Overlay(props: OverlayProps, ref: DOMRef<HTMLDivElement>) {
     return null;
   }
 
+  // need to put position relative on here to address if it has a sibling with a z-index and some position set, like static in our docs
+  // see https://www.freecodecamp.org/news/4-reasons-your-z-index-isnt-working-and-how-to-fix-it-coder-coder-6bc05f103e6c/
+  // still need to beat the sibling's z-index, so 999 for now...
   let contents = (
-    <Provider ref={ref} UNSAFE_style={{background: 'transparent', isolation: 'isolate'}}>
+    <Provider ref={ref} UNSAFE_style={{background: 'transparent', isolation: 'isolate', position: 'relative', zIndex: 999}}>
```

Comment:
999 ok do people think? or should I go much higher? or much lower? what's the likelihood someone will have something similar to what we have in our docs where sibling z-index is a concern?